docs/src/getting-started/basics.md

```diff
@@ -46,39 +57,174 @@ let gas_limit = 1_000_000;
 let byte_price = 0;
 
 // Deploy the contract
-let contract_id = Contract::deploy( & compiled, provider, wallet,
+let contract_id = MyContract::deploy(&compiled, provider, wallet,
 TxParameters::new(gas_price, gas_limit, byte_price)).await.unwrap();
 ```
 
-Alternatively, if you want to launch a local node for every deployment, which is usually useful
-for smaller tests where you don't want to keep state between each test, you can
-use `Provider::launch(Config::local_node())`:
+## Calling and configuring contract calls
+
+Once you've deployed your contract, as seen in the previous section, you'll likely want to call contract methods and configure some parameters such as gas price, byte price, gas limit, and forward coins in your contract call.
+
```

Comment:
The `contract_instance` seems to appear out of nowhere, maybe include something like:

````suggestion

Start by creating an instance of `MyContract`:

```Rust
let contract_instance = MyContract::new(contract_id.to_string(), provider, wallet);
```
````